app/views/course/duplications/show.html.slim

```diff
@@ -1,15 +1,17 @@
 - add_breadcrumb :show
 = page_header t('.header')
 
+- new_start_at = 1.day.from_now.midnight + current_course.start_at.seconds_since_midnight.seconds
```

Comment:
This doesn't solve the issue where the course start_at timing is not on the hour right? eg. 11.23am, or 4.56pm.